xls/examples/BUILD

```diff
@@ -899,3 +899,28 @@ filegroup(
     srcs = glob(["*.x"]),
     visibility = ["//xls:xls_internal"],
 )
+
+xls_dslx_library(
+    name = "passthrough_dslx",
+    srcs = [
+        "passthrough.x",
+    ],
+)
+
+xls_dslx_test(
+    name = "passthrough_dslx_test",
```

Comment:
currently other tests also specify `srcs` (even if I agree it's redundant)

Reply:
changed